packages/fast-components-react-msft/src/pivot/examples.data.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import { Pivot, PivotHandledProps, PivotManagedClasses, PivotProps } from "./index";
+import { TabsItem } from "@microsoft/fast-components-react-base";
+import schema from "./pivot.schema.json";
+import Documentation from "./.tmp/documentation";
+import { ComponentFactoryExample } from "@microsoft/fast-development-site-react";
+
+const pivotManagedClasses: PivotManagedClasses = {
+    managedClasses: {
+        pivot: "pivot-class",
+        pivot_tabList: "pivot_tabList-class",
+        pivot_tab: "pivot_tabclass",
+        pivot_tab__active: "pivot_tab__active-class",
+        pivot_tabPanels: "pivot_tabPanels-class",
+        pivot_tabPanel: "pivot_tabPanel-class",
+        pivot_tabPanel__hidden: "pivot_tabPanel__hidden-class",
+        pivot_tabPanelContent: "pivot_tabPanelContent-class",
+    },
+};
+
+function renderPivot(pivotTitle: string): (className?: string) => React.ReactNode {
+    return (className?: string): React.ReactNode => (
+        <div className={className}>{pivotTitle}</div>
+    );
+}
+
+function renderPivotContent(
+    pivotContent: string
+): (className?: string) => React.ReactNode {

```

Comment:
I believe we no longer should have functional render props code within examples. Take a look at how carousel does this. @janechu for guidance